signer/gpg2/README.rst

```diff
@@ -0,0 +1,102 @@
+PGP Signing with GPG2
+=====================
+
+.. sectnum::
+.. contents:: Table of Contents
+
+This signer implements the Pretty Good Privacy signature format. It
+accepts data on the `/sign/data` interface and returns armored
+detached signatures like the `pgp` signer.
+
+Unlike the `pgp` signer `gpg2` supports loading private keys with
+passphrases exported with the `unsupported and non-standard gnu-dummy
+S2K algorithm`_ and signing with subkeys. However, it does require
+that a gpg2 version > 2.1 be installed.
+
```

Comment:
please add: `gpg2 supports hesoteric key formats that the pgp signer doesn't understand, but it comes at the cost of writing private keys to disk in order to shell out to the gnupg binary. The pgp signer will keep keys in memory at all times, which is more secure. Try using it first and only switch to gpg2 if needed.`